go/install/install_osx.go

```diff
@@ -557,6 +567,96 @@ func InstallKBFS(context Context, binPath string, force bool, timeout time.Durat
 	return nil
 }
 
+// Return where the NativeMessaging host manifest lives on this platform.
+func kbnmManifestPath(u *user.User) string {
```

Comment:
Go function comments are supposed to start with method name `// kbnmManifestPath returns where ...`